docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+#https://docs.docker.com/registry/deploying/#deploy-your-registry-using-a-compose-file
+services:
+  nginx:
+    image: "nginx:alpine"
+    ports:
+      - 5043:443
+    restart: unless-stopped
+    links:
+      - registry:registry
+    volumes:
+      - ./auth:/etc/nginx/conf.d
+      - ./auth/nginx.conf:/etc/nginx/nginx.conf:ro
+  
+  # SSL certificate management for nginx
+  certbot:
+    image: certbot/certbot
+    container_name: 'certbot'
+    volumes:
+      - ../../certbot/conf:/etc/letsencrypt
```

Comment:
Similar question here, we need to determine our local path

Reply:
Maybe somewhere in `/etc`, like `/etc/letsencrypt`?